src/components/TodoPanel.js

```diff
@@ -75,12 +88,12 @@ export default function TodoPanel({setTodosAmount}) {
             <CreateTodoModal isOpen={isTodoModalOpen} closeCreateTodo = {()=> setIsTodoModalOpen(false)} addTodo={addTodo}/>
             <div>
                 <div className = 'search-todo-panel'>
-                    <input className = 'search-todo-input' placeholder='enter a phrase that you are searching for'></input>
-                    {/* <i className="fas fa-search fa-1g"></i> */}
-                    <FontAwesomeIcon icon = {faSearch} className='fa-search'/>
+                    <input className = 'search-todo-input' placeholder='enter a phrase that you are searching for'  ref = {phrase} onChange = {searchTodos}></input>
+                    
+                    {toSearch? <FontAwesomeIcon icon = {faTimes}  className='fa-times' onClick = {cleanInput}/> : <FontAwesomeIcon icon = {faSearch} className='fa-search'/> }
```

Comment:
search icon has no functionality. I see two possibilities:
1. display no icon as long as there is no input
2. display the search icon and when the user clicks on it, while there is empty input, display a tooltip which says something like "please insert a query".

Reply:
it will stay as we discussed